extensions/positron-python/python_files/positron/positron_ipykernel/patch/bokeh.py

```diff
@@ -0,0 +1,47 @@
+#
+# Copyright (C) 2024 Posit Software, PBC. All rights reserved.
+# Licensed under the Elastic License 2.0. See LICENSE.txt for license information.
+#
+
+from logging import Logger
+
+from ..session_mode import SessionMode
+
+MIME_TYPE_POSITRON_WEBVIEW_FLAG = "application/positron-webview-load.v0+json"
+
+
+def handle_bokeh_output(session_mode: SessionMode, logger: Logger) -> None:
```

Comment:
Nit: In other places we've used a module-level logger which should be fine here too. You can add this above, and remove the `logger` arg:

```python
import logging

logger = logging.getLogger(__name__)
```

Reply:
Makes sense. I passed it in so the logger would appear from the `positron_ipykernel` module but it's obviously easier to do that!